build/dependencies.yaml

```diff
@@ -209,12 +206,6 @@ dependencies:
       match: registry.k8s.io\/pause:\d+\.\d+
     - path: cmd/kubeadm/app/constants/constants.go
       match: PauseVersion\s+=
-    - path: cmd/kubeadm/app/util/template_test.go
-      match: validTmpl\s+=
-    - path: cmd/kubeadm/app/util/template_test.go
-      match: validTmplOut\s+=
-    - path: cmd/kubeadm/app/util/template_test.go
-      match: doNothing\s+=
```

Comment:
these are not needed, unit tests were bound to a version, where they don't need to.

cmd/kubeadm has a couple of side cleanups in this PR to not have the image version in tests:
- flags_tests.go
- runtime_test.go